web/tests/narrow_activate.test.js

```diff
@@ -1,292 +0,0 @@
-"use strict";
-
-const assert = require("node:assert/strict");
-
-const {mock_esm, set_global, zrequire} = require("./lib/namespace");
-const {run_test, noop} = require("./lib/test");
-const $ = require("./lib/zjquery");
```

Comment:
Yeah in theory a good node test for this code path would be good, but the coverage of actual logic is quite poor (better replaced with unit tests for individual functions like `maybe_add_local_messages` if we wanted to invest time in it) , and it's so darn much mocking.